### PR TITLE
IW Support Custom Functions

### PR DESCRIPTION
**Pull request by khanm-ibl**
IW Support Custom Functions:
- iwSendTransaction
- iwGetAccounts
- iwSignMessage
- iwSendRaw

### CODE REVIEW

File: packages/core/src/index.ts [hunk 1]
```diff
@@ -561,6 +561,109 @@ class Connector implements IConnector {
     }
   }
 
+  /**
+   * IW send transaction
+   * @param platform 
+   * @param params 
+   */
+  public async iwSendTransaction (platform: string ,params: any) {
```
Comment (taipa-ibl) [inline packages/core/src/index.ts]: @khanm-ibl 
Please format your new code before submit.

File: packages/core/src/index.ts [hunk 2]
```diff
@@ -561,6 +561,109 @@ class Connector implements IConnector {
...
+      throw error
+    }
+  }
+
+  /**
+   * IW get wallet address by platform
+   * @param platform 
+   * @param options 
+   */
+  public async iwGetAccounts (platform: string , options: any) {
```
Comment (taipa-ibl) [inline packages/core/src/index.ts]: @khanm-ibl 
Did this methods return multi accounts ? Or just single account ? The base document have 2 functions is getAddress and getAddresses, please check more details about that.

File: packages/core/src/index.ts [hunk 3]
```diff
@@ -561,6 +561,109 @@ class Connector implements IConnector {
...
+    })
+
+    try {
+      const result = await this._sendCallRequest(request)
+      return result
+    } catch (error) {
+      throw error
+    }
+  }
+
```
Comment (taipa-ibl) [inline packages/core/src/index.ts:693]: @khanm-ibl 
Maybe we need a custom function like: iwCustomFunction(method: string,  platform: string, data: any).

**Review by taipa-ibl — COMMENTED** [review]
@khanm-ibl 
I have already review your script, please have a looks. Thank you

File: packages/core/src/index.ts (hunk 2 above)
Comment (khanm-ibl) [reply packages/core/src/index.ts]: This function will cover 2 case
- options.currentWalletOnly === true // get current wallet's address only
- options.currentWalletOnly is empty or value is false : return all by default
@taipa-ibl let me know your idea, thank!

File: packages/core/src/index.ts (hunk 1 above)
Comment (khanm-ibl) [reply packages/core/src/index.ts]: @taipa-ibl sure!

File: packages/core/src/index.ts (hunk 3 above)
Comment (khanm-ibl) [reply packages/core/src/index.ts:693]: @taipa-ibl okay a

File: packages/core/src/index.ts (hunk 2 above)
Comment (taipa-ibl) [reply packages/core/src/index.ts]: It's good, but the options should be
`options.allWallet === true` for get all wallet accounts
`options.allWallet === false or empty or undefined` for get current wallet only.

**Pull request merged**